doc/libpmemobj/pmemobj_tx_begin.3.md

```diff
@@ -412,6 +418,19 @@ If **pmemobj_tx_set_user_data**() was not called for a current transaction,
 during **TX_STAGE_WORK** or **TX_STAGE_ONABORT** or **TX_STAGE_ONCOMMIT** or
 **TX_STAGE_FINALLY**.
 
+**pmemobj_tx_set_abort_on_failure**() enables or disables automatic transaction
+abort in case **pmemobj_tx_add_rage**, **pmemobj_tx_add_rage_direct**, **pmemobj_tx_alloc**,
```

Comment:
We will have to remember to update this list. I'm not sure if it wouldn't make more sense to just write all transactional functions except for X, Y and Z.